packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -77,6 +75,7 @@ class ModelClient:
 
     def __init__(self,
                  url=None,
+                 base_url=None,
```

Comment:
```suggestion
                 base_url,
                 path_prefix,
```
Specifying both `base_url` and `url` seems a bit redundant. I'd suggest `base_url` + `path_prefix` instead.

`base_url` - ODAHU API URL
`path_prefix` - path relative to base url where the model is deployed, like `/model/my-cool-model`. This can be taken from ModelRoute status by user. 

So to get a full model URL you can do `urljoin(base_url, path_prefix)`.